src/module.ts

```diff
@@ -40,15 +39,15 @@ export default defineNuxtModule({
       `).join('\n')
     }).dst
 
-    const runtimeDir = await resolver.resolvePath(isNuxt2() ? './runtime/vue2' : './runtime/vue3')
+    const runtimeDir = await resolver.resolvePath('./runtime')
```

Comment:
For simply resolving relative path `resolver.resolve` can be used. Will likely deprecate and rename `resolvePath` to clear it's purpose :D